lib/aptible/cli/agent.rb

```diff
@@ -81,10 +84,20 @@ def version
       option :lifetime, desc: 'The duration the token should be valid for ' \
                               '(example usage: 24h, 1d, 600s, etc.)'
       option :otp_token, desc: 'A token generated by your second-factor app'
+      option :security_key, desc: 'Use Security Key authentication if available'
       def login
```

Comment:
I thought it wasn't in `subcommands` because it was a singe command unlike `apps`, `apps:scale`, etc. but `domains` is a single command and it's in `subcommands` so it may just be an artifact of how the CLI used to be structured. I think moving it and, possibly, `version` is worth considering but not as part of this PR.

Reply:
:+1: a later cleanup PR makes sense